### ISSUE TITLE
Add support for custom host metadata

### ISSUE THREAD

**Issue by agardnerIT** [Dynatrace/Dynatrace-OneAgent-Ansible#13]
Add support for creating custom metadata via the `hostcustomproperties.conf` file

**Comment by jonathansd1** [comment]
Breathing new life into this issue -- from @kristofre : 
```
We need to check if our Ansible/Chef/Puppet modules currently have the ability to
set meta data, name and tags for the host (they are stored in 3 config files).
If not, I suggest that we should look at adding those in to support our usecases

**Comment by luisescobar89** [comment]
added in fixed in v1.0.7 release
